rails/Gemfile.lock

```diff
@@ -374,7 +374,7 @@ DEPENDENCIES
   webpacker (~> 5.0)
 
 RUBY VERSION
-   ruby 2.7.7p221
+   ruby 2.7.8p225
```

Comment:
Looks like this updated here finally https://github.com/Terrastories/terrastories/pull/910#issuecomment-1528829225